src/app/src/main/kotlin/org/imperial/mrc/hint/db/DataSourceConfig.kt

```diff
@@ -0,0 +1,18 @@
+package org.imperial.mrc.hint.db
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.jdbc.DataSourceBuilder
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import javax.sql.DataSource
+
+@Configuration
+open class DataSourceConfig
+{
+    @Bean
+    @ConfigurationProperties(prefix="spring.datasource")
```

Comment:
This will pull in datasource configuration from the application properties files, from keys with the spring.datasource prefix